Makefile

```diff
@@ -188,7 +198,10 @@ push-plugin: plugin-container
 push-datamgr: datamgr-container
 	docker push $(DATAMGR_IMAGE):$(VERSION)
 
-push: push-datamgr push-plugin
+push-backup-driver: backup-driver-container
+	docker push $(BACKUPDRIVER_IMAGE):$(VERSION)
+
+push: push-datamgr push-plugin push-backup-driver
 
 push-pp:
 	$(MAKE) push-plugin LOCALMODE=true VERSION=$(VERSION)-pp
```

Comment:
as we will make change to plugin, we will have to push `backup-driver` for power protect support.